CONTRIBUTING.md

```diff
@@ -0,0 +1,99 @@
+# Contributing to reek
+
+We welcome any and all contributions to reek! [Report
+issues](https://github.com/troessner/reek/issues) with
```

Comment:
I would remove the "Report issues" sentence completely since it feels unnecessary and distracting given what we write down below.